modules/@apostrophecms/schema/ui/apos/logic/AposInputArray.js

```diff
@@ -28,30 +28,81 @@ export default {
     const next = this.getNext();
     // this.schema is a computed property and is not available in data, that's why we use this.field.schema here instead
     const items = modelItems(next, this.field, this.field.schema);
-
     return {
       next,
       items,
+      isDragging: false,
       itemsConditionalFields: Object
-        .fromEntries(items.map(({ _id }) => [ _id, getConditionTypesObject() ]))
+        .fromEntries(items.map(({ _id }) => [ _id, getConditionTypesObject() ])),
+      emptyWhenIcon: this.field?.whenEmpty?.icon || 'text-box-multiple-icon',
+      emptyWhenLabel: this.field?.whenEmpty?.label || 'apostrophe:noItemsAdded'
     };
   },
   computed: {
+    isInlineTable() {
+      return this.field.style === 'table' && this.field.inline;
+    },
+    isInlineStandard() {
+      return this.field.style !== 'table' && this.field.inline;
+    },
+    isDraggable() {
+      if (this.field.draggable === false) {
+        return false;
+      }
+      if (this.field.readOnly) {
+        return false;
+      }
+      if (this.next.length > 1) {
+        return true;
+      }
```

Comment:
You could remove this last if statement since i'll fallback on true anyway.
Or maybe you want the opposite, returning false when the length is equal to 1 OR 0?

Reply:
should have been `this.next.length <= 1 ? return false`